libs/langchain/langchain/callbacks/openai_info.py

```diff
@@ -6,27 +6,31 @@
 
 MODEL_COST_PER_1K_TOKENS = {
     # GPT-4 input
+    "gpt-4-1106-preview": 0.01,
     "gpt-4": 0.03,
     "gpt-4-0314": 0.03,
     "gpt-4-0613": 0.03,
     "gpt-4-32k": 0.06,
     "gpt-4-32k-0314": 0.06,
     "gpt-4-32k-0613": 0.06,
     # GPT-4 output
+    "gpt-4-1106-preview": 0.03,
```

Comment:
this is missing the `"-completion"`, otherwise you are only overwriting the entry above.